user_funcs.py

```diff
@@ -116,3 +117,246 @@ def login(conn) -> tuple[str, int]:
                 return results[0]
 
             print("Username or password incorrect!")
+#THis whole function does work correctly
+def create_collection(conn, user_ID) -> None:
+    """
+    creates a new collection for the user and puts the new collection
```

Comment:
NIT: (Short for nitpick. I'll use this for minor things that shouldn't effect functionality)
```suggestion
    Creates a new collection for the user and puts the new collection
```